binding/Binding/SKMatrix.cs

```diff
@@ -444,6 +535,78 @@ public static void PostConcat (ref SKMatrix target, ref SKMatrix matrix)
 			}
 		}
 
+		// *Scale
+		public void PreScale(float sx, float sy)
+		{
+			SKMatrix o;
+			fixed (SKMatrix* t = &this)
+			{
+				SkiaApi.sk_matrix_pre_scale(&o, t, sx, sy);
+			};
+			setFrom(ref o);
+		}
+
+		public void PreScale(float sx, float sy, float px, float py)
+		{
+			SKMatrix o;
+			fixed (SKMatrix* t = &this)
+			{
+				SkiaApi.sk_matrix_pre_scale_with_pivot(&o, t, sx, sy, px, py);
+			};
+			setFrom(ref o);
+		}
+
+		public void PostScale(float sx, float sy)
+		{
+			SKMatrix o;
+			fixed (SKMatrix* t = &this)
+			{
+				SkiaApi.sk_matrix_post_scale(&o, t, sx, sy);
+			};
+			setFrom(ref o);
+		}
+
+		public void PostScale(float sx, float sy, float px, float py)
+		{
+			SKMatrix o;
+			fixed (SKMatrix* t = &this)
+			{
+				SkiaApi.sk_matrix_post_scale_with_pivot(&o, t, sx, sy, px, py);
+			};
+			setFrom(ref o);
+		}
+
+
+		public void PreTranslate(float dx, float dy)
+		{
+			SKMatrix o;
+			fixed (SKMatrix* t = &this)
+			{
+				SkiaApi.sk_matrix_post_translate(&o, t, dx, dy);
```

Comment:
shouldn't it read `sk_matrix_pre_translate()` here?

Reply:
Not sure, havnt worked on it in ages